README.md

```diff
@@ -210,6 +210,12 @@ Hints are used in two ways, if the server is configured to use them - see [below
   If hints are available, then the resulting match is checked against the list of hints and
   flagged with a `hintMismatch` issue if the match does not correspond to the hint.
 
+#### Loose matches
```

Comment:
The word `loose` was not obvious to me as it's a loose search using the supplied scientific name, until I read the code:) 
Could be a bit unclear for other users as well?

Reply:
Updated README.md